test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -127,6 +130,85 @@ class RepresenteeControllerSpec
     sessionWithFillingOutReturn(Some(answers), Some(representativeType.merge), subscribedDetails)
 
   "RepresenteeController" when {
+    "handling change contact name" must {
+      "display the page" when {
+        def performAction(): Future[Result] = controller.changeContactName()(FakeRequest())
+
+        def test(
+          sessionData: SessionData,
+          expectedTitleKey: String,
+          expectedBackLink: Call,
+          expectedPrepopulatedValue: ContactName
```

Comment:
it doesn't look like you're using this value

Reply:
Done.